codejail_service/tests/test_startup_check.py

```diff
@@ -35,9 +35,9 @@ def responses(math=DEFAULT, disk=DEFAULT, child=DEFAULT):
 
     This is intended to be used in mocking out safe_exec in the exact
     sequence of calls that the startup safety check function
-    makes. Each value in the list should either be a return value of
-    safe_exec or an Exception instance. (Anything that is compatible with
-    the behavior of the side_effect param in unittest.mock.)
+    makes. Each value in the list must be a return value of
+    safe_exec -- that is, a 2-tuple of the returned globals and an emsg (error
+    string, or None).
```

Comment:
Minor change to make it more clear this isn't the tuple, although I know the "or" messes that up.
```suggestion
    string or None).
```

Reply:
Updated a bit differently than that, see what you think.